### PR TITLE
[LOC] Make OSGi EventListener as powerful as LicensePackIssueListener

### PR DESCRIPTION
**Pull request by HannesWell**
Fixes https://github.com/eclipse-passage/passage/issues/1399

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.licenses.core/src/org/eclipse/passage/loc/internal/licenses/core/IssueFloatingLicense.java [hunk 1]
```diff
@@ -120,6 +124,7 @@ private ServiceInvocationResult<List<Path>> persist(EObject target, LicensedProd
 		Path lic;
 		try {
 			lic = new PersistedDecoded(folder, target).write(decrypted);
+			events.sendEvent(new OperatorLicenseEvents().decodedIssued(lic.toString()));
```
Comment (HannesWell) [inline bundles/org.eclipse.passage.loc.licenses.core/src/org/eclipse/passage/loc/internal/licenses/core/IssueFloatingLicense.java:127]: Btw. what do you think about passing the `Path` object directly, here and below?

**Comment by eparovyshnaya** [comment]
Closing this MR as it does not solve the underlying problem of #1399 issue. 

See [the comment](https://github.com/eclipse-passage/passage/issues/1399#issuecomment-2208494122) to get the plan of works on the issue we are going to entertain.

**Comment by HannesWell** [comment]
> See [the comment](https://github.com/eclipse-passage/passage/issues/1399#issuecomment-2208494122) to get the plan of works on the issue we are going to entertain.

From your comment I understand that this does not completely solve #1399, but it at least brings the OSGi `EventListener` variant on par. From my point of view, this is at least half of the solution.
The missing part, if I understand your intention correctly, is to remove `LicensePackIssueListener` and migrate existing implementations to the OSGi event mechanism, isn't it?
Or do I miss something?

**Comment by eparovyshnaya** [comment]
This MR is harmful: it synchs event processing. Even if it turns to event posting, it is insignificant as it is partial in a place that does not help the product.

**Comment by eparovyshnaya** [comment]
Synching what was designed to be asynch is harmfull, for one, because there can be long-running operations hanging on those events.

**Comment by HannesWell** [comment]
> Synching what was designed to be asynch is harmfull, for one, because there can be long-running operations hanging on those events.

OK, if listeners are designed to be asynch and run long, then I agree. I assumed that they should be short-running anyways.
In that case I suggest to introduce a second, event type that is fired synchronously.

**Pull request closed without merging**